extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/common/GooglePhotosImportUtils.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2022 The Data Transfer Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.datatransferproject.datatransfer.google.common;
+
+import com.google.common.base.Strings;
+
+public class GooglePhotosImportUtils {
```

Comment:
```suggestion
// TODO(#yyyyyyy) consider porting this logic to a transmogrify config (and thus deleting this class).
public class GooglePhotosImportUtils {
```

Reply:
ah, I see you [already discussed](https://github.com/google/data-transfer-project/pull/1134#discussion_r946667754) this exact thing -- yeah a TODO-tracker is sufficient I think.